src/commands/SuspendCommand.ts

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2024 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {Mjolnir} from "../Mjolnir";
+import {RichReply} from "matrix-bot-sdk";
+
+export async function execSuspendCommand(roomId: string, event: any, mjolnir: Mjolnir, parts: string[]) {
+    const victim = parts[2];
```

Comment:
this is almost certainly a copy/paste, but we should find a different term to describe the affected user. "victim" isn't appropriate anymore (and maybe never was, honestly).

Not something for this PR.